tests/common.go

```diff
@@ -3566,6 +3588,37 @@ func CreateSourceAndDestClusters(orgID string, cloudName string, uid string, ctx
 	return nil
 }
 
+// DumpClusterConfig creates source and destination cluster
+
+func AddApplicationCluster(kubeconfig string, orgID string, cloudName string, uid string, ctx context1.Context) error {
```

Comment:
How about making `kubeconfig` an optional parameter in `CreateSourceAndDestinationCluster`, similar to `CreateCloudCredential`? It would make maintenance easier. Please suggest @mkoppal-px